assets/src/edit-story/app/prepublish/guidance/media.js

```diff
@@ -145,15 +145,16 @@ export function videoElementLength(element) {
  * @return {Guidance|undefined} The guidance object for consumption
  */
 export function videoElementOptimized(element = {}) {
-  const idResource = element.resource?.id;
-  const idOrigin = idResource?.toString().split(':')?.[0];
-  const isCoverrMedia = idOrigin === 'media/coverr';
   const videoArea =
     (element.resource?.height ?? 0) * (element.resource?.width ?? 0);
   const isLargeVideo =
     videoArea >= VIDEO_SIZE_THRESHOLD.WIDTH * VIDEO_SIZE_THRESHOLD.HEIGHT;
 
-  if (!isCoverrMedia && isLargeVideo && !element.resource?.isOptimized) {
+  if (
+    isLargeVideo &&
+    !element.resource?.local &&
+    !element.resource?.isOptimized
+  ) {
```

Comment:
I'd just write:

```js
if (element.resource?.local) {
  return undefined;
}

if (element.resource?.isOptimized) {
  return undefined;
}

if (isLargeVideo) {
 return {...}
}
```

Reply:
Why give there more execution paths and early returns. I think this makes the code a little overly verbose

Reply:
To reduce complexity / the number of execution paths and make it more readable.

Reply:
I don't personally find this line that hard to read...

Reply:
I like it. Keeps the conditional from getting to bloated and ugly as we add more checks to it. agree that rn with the amount of checks there aren't as many benefits but 🤷